### DISCUSSION TITLE
Replit

### DISCUSSION THREAD

**Discussion by Gingeh** [catppuccin/catppuccin#1899] [Port Requests]
### Will you make the port?

- [X] Yes, I'm willing to make the port

### How can the requested port be themed?

Replit now officially supports custom themes, which can be created using a very helpful editor at https://replit.com/themes. 
![Replit theme editor](https://user-images.githubusercontent.com/39150378/215679205-cbf9609c-acb3-4912-9bdb-cc6ff2a7716f.png)

### Has the requested port been themed already?

There are several existing "catppuccin" themes, however, it would be good to create a set of official themes.
![Existing catppuccin themes](https://user-images.githubusercontent.com/39150378/215678574-4e19abe1-f8e7-4824-88cf-02e03d609d13.png)

### Any additional comments?

We have a few options:
1. We contact one of the users that already made a theme and try to make it official
2. We create a set of themes from scratch

Also, should we use a catppuccin branded account or just a personal account?

**Comment by Gingeh** [comment]
Here is my initial attempt at a mocha theme..
![Replit mocha](https://user-images.githubusercontent.com/39150378/215714519-fff9ccc3-9227-4ca7-b0f5-f8787e3431c3.png)

..and a macchiato theme.
![Replit macchiato](https://user-images.githubusercontent.com/39150378/215716761-6218015c-2f70-4ccf-a040-56cd6dfe9ec4.png)

**Reply by Gingeh** [reply]
There are some tokens that I don't seem to be able to theme (e.g. the derive attribute and println macro in the above screenshots)

**Comment by sgoudham** [comment]
We should definitely have an official one! I'd like it to come from a Catppuccin™️ account if possible.
I really should get publishing that editor style guide... 😅

**Reply by Gingeh** [reply]
What would the process be for creating a Catppuccin:tm: account? (shared email/password?)

**Reply by Gingeh** [reply]
I've created an account: https://replit.com/@Catppuccin

**Comment by Gingeh** [comment]
I have created a theme for each flavor: https://replit.com/themes?authorId=20252294

(The latte theme probably needs more work)

**Comment by Gingeh** [comment]
I've created a port repo for the replit themes: https://github.com/Gingeh/replit

**Reply by sgoudham** [reply]
Thanks! Transferred to #1903
